$NetBSD: patch-al,v 1.1 2011/01/02 23:53:13 schnoebe Exp $

Add NetBSD to the list of resolvers that support threading, having a
"modern" resolver API.

--- iris/src/jdns/jdns_sys.c.orig	2009-07-26 16:02:12.000000000 -0500
+++ iris/src/jdns/jdns_sys.c
@@ -679,7 +679,7 @@ static jdns_dnsparams_t *dnsparams_get_u
 	return params;
 }
 
-#if defined(__GLIBC__) && ((__GLIBC__ > 2) || ((__GLIBC__ == 2) && (__GLIBC_MINOR__ >= 3)))
+#if defined(__GLIBC__) && ((__GLIBC__ > 2) || ((__GLIBC__ == 2) && (__GLIBC_MINOR__ >= 3))) || defined(JDNS_OS_NETBSD)
 # define JDNS_MODERN_RES_API
 #endif
 
